lighthouse-core/scripts/lantern/constants.js

```diff
@@ -1,16 +1,195 @@
 /**
  * @license Copyright 2018 Google Inc. All Rights Reserved.
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 'use strict';
 
 const path = require('path');
 
 /* eslint-disable max-len */
 
+/**
+ * @typedef LanternSiteDefinition
+ * @property {string} url
```

Comment:
(as followup, if at all) wonder if we should just make a `.d.ts` file. then we can do `Lantern.Evaluation, Lantern.SiteDefinition`, etc. and TS is easier to write than JSDOC

Reply:
agreed, left a comment 👍